docs/modules/ROOT/pages/jet-job-configuration.adoc

```diff
@@ -20,7 +20,9 @@ Below are the configuration options for the `JetJob` resource. You can find more
 |HazelcastResourceName defines the name of the Hazelcast resource.
 
 |state
-|State is used to manage the job state.
+|State is used to manage the job state. The default value is 'Running'.
+
+WARNING: The value of `state` field must be `Running` when the JetJob object is created for the first time.
```

Comment:
I would have added this as a regular documentation sentence. This is not a warning, but a requirement

Reply:
So, should we remove it?